src/test/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCallerTest.scala

```diff
@@ -617,6 +618,12 @@ class VanillaUmiConsensusCallerTest extends UnitSpec with OptionValues {
     s2.cigar.toString shouldBe "40M20I40M"
   }
 
+  it should "except when the reads do not have base qualities and a minimum base quality is not given" in {
```

Comment:
The exception will occur regardless of minimum base quality:

```suggestion
  it should "except when the reads do not have base qualities" in {
```